Dockerfile

```diff
@@ -27,11 +27,20 @@ RUN apt-get install -y \
     python-pip \
     openjdk-7-jre-headless
 
+# Install Java.
+RUN \
+    apt-get update && \
+    apt-get install -y openjdk-7-jdk && \
```

Comment:
can we merge this with the previous RUN? this looks like a part of 'Install runtime packages.